src/components/BasicInfo.tsx

```diff
@@ -15,12 +15,16 @@ export const BasicInfo = ({ pet }: any) => {
     >
       <div
         className="rounded-xl shadow-md aspect-auto overflow-hidden"
-        style={{ gridColumn: '1/2' }}
+        style={{ gridColumn: '1/2', maxWidth: '200px' }}
       >
-        <img className="object-cover" src={pet.img} />
-        {/* TODO: use <Image> */}
-        {/* problem: unconfigured host, look into that */}
-        {/* <Image src={pet.img} alt={pet.name} width='100' height='200' /> */}
+        <div className="relative w-full h-full">
```

Comment:
What's the reason behind using relative positioning?

Observation:
pet image is not being displayed on small screens